### PR TITLE
bump kubernetes version for common image

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/common/issues/1251

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Bumping version of PyYaml (which depends on Kubernetes, which depends on openshift) for CVE_2020_1747

### CODE REVIEW

**Review by harshad16 — CHANGES_REQUESTED** [review]
please revert pre-commit chnages

File: Pipfile [hunk 1]
```diff
@@ -8,10 +8,10 @@ argo-workflows = ">=3.5,<4"
 daiquiri = "*"
 requests = "*"
 sentry-sdk = "*"
-kubernetes = "~=11.0.0"  # required pyyaml>=3.12
-openshift = "==0.11.0"
+kubernetes = ">=12.0.1"  # required pyyaml>=5.3.1
+openshift = "==0.12.0"
```
Comment (harshad16) [inline Pipfile:12]:
```suggestion
openshift = "~=0.12.0"
```
How about this https://github.com/openshift/openshift-restclient-python/tree/v0.12.1
It has some changes: https://github.com/openshift/openshift-restclient-python/commits/v0.12.1

**Review by harshad16 — CHANGES_REQUESTED** [review]
All lgtm
can you please rebase your PR that would get to 1 commit 
and also one suggestion.

**Review by fridex — APPROVED** [review]
Changes look good to me. I've also tried to verify that the common library is still able to schedule workfload in the cluster. No issues observed, but we should keep an eye on this change. 👍🏻

**Pull request merged**